src/i18n/languages/ro.json

```diff
@@ -28,6 +28,18 @@
   "language-english": "Engleza",
   "language-romanian": "Romana",
   "edit": "Editat",
+  "signUp.title": "Creeaza un cont",
+  "signUp.firstName": "Completati cu prenumele dvs",
+  "signUp.lastName": "Completati cu  numele dvs",
+  "signUp.email.valid": "Adaugati un email valid",
+  "signUp.email.required": "Emailul este necesar",
+  "signUp.password.valid": "Parola trebuie sa fie mai lunga de 8 caractere",
```

Comment:
This looks like the key should be named `min` or `minimum-characters` instead of `valid`.